src/app/technical-record-search/technical-record-search.component.html

```diff
@@ -25,14 +27,22 @@ <h1 class="govuk-heading-xl">Search for a technical record</h1>
               </div>
             </span>
           </div>
-
-          <input #q
+          <input #searchIdentifier
             [ngClass]="{'govuk-input': true, 'govuk-input--error': (searchError$ | async) ? (searchError$ | async)!=null || (searchError$ | async)!=undefined : false }"
             id="searchIdentifier" name="searchIdentifier" type="text" aria-describedby="name-error">
         </div>
+      
+        <div class="govuk-form-group">
+          <label class="govuk-label govuk-!-font-weight-bold">
+            Search criteria
+          </label>
+          <select class="govuk-select" id="test-search-criteria" #searchCriteria>
+            <option *ngFor="let searchCriteria of searchCriteriaOptions" [value]="searchCriteria">{{ searchCriteria }}</option>
+          </select>
+        </div>
 
-        <a role="button" (click)="searchTechRecords(q.value)" draggable="false"
-          class="govuk-button govuk-button--start mt-1" data-module="govuk-button">
+        <a role="button" (click)="searchTechRecords(searchIdentifier.value, searchCriteria.value)" draggable="false" class="govuk-button govuk-button--start mt-1"
```

Comment:
no custom CSS please follow this link https://design-system.service.gov.uk/styles/spacing/